src/XenCert/StorageHandlerUtil.py

```diff
@@ -73,29 +73,20 @@
 def _init_adapters():
     # Generate a list of active adapters
     ids = scsiutil._genHostList(ISCSI_PROCNAME)
-    util.SMlog(ids)
+    util.SMlog("Host ids: %s" % ids)
     adapter = {}
     for host in ids:
         try:
-            # For Backward compatibility 
-            if hasattr(iscsilib, 'get_targetIQN'):
-                targetIQN = iscsilib.get_targetIQN(host)
-            else: 
-                targetIQN = util.get_single_entry(glob.glob(\
-                   '/sys/class/iscsi_host/host%s/device/session*/iscsi_session*/targetname' % host)[0])
-
             if hasattr(iscsilib, 'get_targetIP_and_port'):
                 (addr, port) = iscsilib.get_targetIP_and_port(host)
             else:
-                addr = util.get_single_entry(glob.glob(\
-                   '/sys/class/iscsi_host/host%s/device/session*/connection*/iscsi_connection*/persistent_address' % host)[0])
-                port = util.get_single_entry(glob.glob(\
-                   '/sys/class/iscsi_host/host%s/device/session*/connection*/iscsi_connection*/persistent_port' % host)[0])
-
-            entry = "%s:%s" % (addr,port)
-            adapter[entry] = host
+                addr = util.get_single_entry(glob.glob(
+                    '/sys/class/iscsi_host/host%s/device/session*/connection*/iscsi_connection*/persistent_address' % host)[0])
+                port = util.get_single_entry(glob.glob(
+                    '/sys/class/iscsi_host/host%s/device/session*/connection*/iscsi_connection*/persistent_port' % host)[0])
+            adapter[host] = (addr, port)
```

Comment:
If this line is not aligned with above (i.e. out of the else branch scope), please add a blank line above it.

should this line be included in the else statement branch?